src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java

```diff
@@ -365,6 +365,25 @@ Integer insertDataset(
             " WHERE d.dac_id = :dacId ")
     Set<DatasetDTO> findDatasetsByDac(@Bind("dacId") Integer dacId);
 
+    /**
+     * Finds all datasets which are assigned to this DAC and which
+     * have been requested for this DAC.
+     *
+     * @param dacId id
+     * @return all datasets associated with DAC
+     */
+    @UseRowReducer(DatasetReducer.class)
+    @SqlQuery("SELECT d.*, k.key, dp.property_value, dp.property_key, dp.property_type, dp.schema_property, dp.property_id, ca.consent_id, c.translated_use_restriction, dar_ds_ids.id as in_use " +
+            " FROM dataset d " +
+            " LEFT JOIN (SELECT DISTINCT dataset_id AS id FROM dar_dataset) dar_ds_ids ON dar_ds_ids.id = d.dataset_id " +
+            " LEFT JOIN dataset_property dp ON dp.dataset_id = d.dataset_id " +
+            " LEFT JOIN dictionary k ON k.key_id = dp.property_key " +
+            " LEFT JOIN consent_associations ca ON ca.dataset_id = d.dataset_id " +
+            " LEFT JOIN consents c ON c.consent_id = ca.consent_id " +
```

Comment:
As of https://github.com/DataBiosphere/consent/pull/1724 (which is merged), I believe you shouldn't have to join on these tables any longer.

Reply:
Ah ... I see `ca.consent_id, c.translated_use_restriction` in the query so we do need those columns. Those two values are of less value now that we store the data use on the dataset. `translated_use_restriction` is actually quite incorrect. It represents the translation of the V1 algorithm. `consent_id` is also useless since its only function was to tie the dataset to the use restriction on a consent. I will file tickets for deprecating these.